ingest/loom.py

```diff
@@ -0,0 +1,61 @@
+"""Module for ingesting Loom files into Firestore
+
+DESCRIPTION
+Module provides extract and transforms function for gene expression data for
+a loom File.
+
+PREREQUISITES
+Must have python 3.6 or higher.
+"""
+
+
+import sys
+
+import loompy
+from gene_data_model import Gene
+import numpy as np
+
+np.set_printoptions(precision=8, threshold=sys.maxsize, edgeitems=1e9)
+
+
+class Loom:
+    def __init__(self, file_path, file_id, study_accession, **kwargs):
+        self.ds = loompy.connect(file_path)
+        self.study_accession = study_accession
+        self.file_id = file_id
+        self.matrix_params = {k: v for k, v in kwargs.items() if v is not None}
+
+    def extract(self):
+        """Pythons iterator protocol. Reads though Loom file and extracts 512 rows at time
+
+        Returns:
+        ------
+            view: < generator LoomView>
+                    A generator of a view corresponding to the current chunk
+        """
+        for (ix, selection, view) in self.ds.scan(axis=0, batch_size=10):
+            yield view
+
+    def transform_expression_data_by_gene(self, view) -> Gene:
```

Comment:
Would this type annotation be more correct?

```suggestion
    def transform_expression_data_by_gene(self, view) -> List[Gene]:
```

Reply:
Addressed in f708656